cvat/apps/engine/views.py

```diff
@@ -1275,24 +1277,75 @@ def content(self, request, pk):
                 'specific_attributes': db_storage.get_specific_attributes()
             }
             storage = get_cloud_storage_instance(cloud_provider=db_storage.provider_type, **details)
-            storage.initialize_content()
-            storage_files = storage.content
-
             manifest_path = request.query_params.get('manifest_path', 'manifest.jsonl')
-            with TemporaryDirectory(suffix='manifest', prefix='cvat') as tmp_dir:
-                tmp_manifest_path = os.path.join(tmp_dir, 'manifest.jsonl')
-                storage.download_file(manifest_path, tmp_manifest_path)
-                manifest = ImageManifestManager(tmp_manifest_path)
+            if not storage.is_object_exist(manifest_path):
```

Comment:
Could we get a case when ``not storage.exists():`` here?

Reply:
Yes, I will add